src/redux/reducers/settings.js

```diff
@@ -43,6 +55,62 @@ export default handleActions(
         ...state,
       };
     },
+
+    [actionTypes.SETTINGS_SET_ACCOUNT_SETTINGS](state, { payload }) {
+      const { userName, timeZone, prefLanguage } = payload;
+      return {
+        ...state,
+        userName,
+        timeZone,
+        prefLanguage,
+      };
+    },
+    [actionTypes.SETTINGS_SET_MOBILE_SETTINGS](state, { payload }) {
+      const { phoneNo, countryCode, countryDialCode } = payload;
+      return {
+        ...state,
+        phoneNo,
+        countryCode,
+        countryDialCode,
+      };
+    },
+    [actionTypes.SETTINGS_SET_CHAT_PREFERENCES_SETTINGS](state, { payload }) {
+      return {
+        ...state,
+        enterAsSend: !state.enterAsSend,
```

Comment:
Do not do this. Send true/false from component

Reply:
Okay